lnrpc/devrpc/driver.go

```diff
@@ -0,0 +1,67 @@
+//go:build dev
```

Comment:
Do we need to update makefile's linter to include files behind build tags, or are they already covered?

Reply:
Good catch, we never covered files with the `dev` tag before. Added in separate commit.